sensing_launch/launch/aip_customized/gnss.launch.xml

```diff
@@ -16,15 +16,15 @@
         <arg name="input_topic_fix" value="ublox/nav_sat_fix" />
         <arg name="input_topic_navpvt" value="ublox/navpvt" />
 
-        <arg name="output_topic_gnss_pose" value="pose" />
-        <arg name="output_topic_gnss_pose_cov" value="pose_with_covariance" />
-        <arg name="output_topic_gnss_fixed" value="fixed" />
+      <arg name="output_topic_gnss_pose" value="pose" />
```

Comment:
Just a comment. There is no standard, but the ideation for XML is a little bit weird.